x-pack/plugins/observability/server/index.ts

```diff
@@ -48,7 +48,7 @@ const configSchema = schema.object({
     }),
     thresholdRule: schema.object({
```

Comment:
Does this `thresholdRule` actually refer to custom therhold rule?

Reply:
This rule was initially called `thresholdRule` and since we will remove the feature flag in the future, we decided not to change the name for minimal disruption.